05/app.ts

```diff
@@ -0,0 +1,44 @@
+let marks: number[] = []; 
+
+function addMark(mark: number): void{
+    if(mark > 10){
+        return
+    }
+    marks.push(mark); 
+}
+
+addMark(12);
+addMark(9);
+addMark(6);
+addMark(6);
+
+function getAverage(): number {
```

Comment:
Zdecydowanie lepiej jest gdy funkcja przyjmuje dane przez parametr, na których ma operować.

PS. Tablica to obiekt, więc będzie to referencja, która pozwoli dodawać kolejne elementy do tablicy.